rosbag2_transport/include/rosbag2_transport/player.hpp

```diff
@@ -120,7 +132,16 @@ class Player : public rclcpp::Node
   /// this method will wait until new element will be pushed to the queue.
   /// \return true if player in pause mode and successfully played next message, otherwise false.
   ROSBAG2_TRANSPORT_PUBLIC
-  bool play_next();
+  virtual bool play_next();
+
+  ROSBAG2_TRANSPORT_PUBLIC
+  void add_key_callback(
+    KeyboardHandlerBase::KeyCode key,
+    const std::function<void()> & cb,
+    const std::string & op_name);
+
+  ROSBAG2_TRANSPORT_PUBLIC
+  void add_keyboard_callbacks();
```

Comment:
should these methods be private/protected? they feel like implementation details